students/ghassan/lesson03/mailroom.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+
+import sys
+
+donors = [
+    ('Jessie', (100, 300, 500)),
+    ('James', (343, 22.11)),
+    ('Ford', (3000, 400))]
+
+
+def prompt():
+    choice = input(
+        'Pick one:\n1) Send a thank you\n2) Create a Report\n3) Quit\n')
+    while choice not in ['1', '2', '3']:
+        choice = input(
+            'Pick one:\n1) Send a thank you\n2) Create a Report\n 3) Quit\n')
+    return choice
+
+
+def send_thankyou():
+    donor_name = input('Enter the donor\'s name: ')
+    if donor_name == 'list':
+        for donor in donors:
+            print(donor[0])
+    elif donor_name not in donors:
+        donors.append((donor_name, ))
```

Comment:
your mailroom program should also have functionality to add donor amount to existing donors (which I don't see here).

Since that functionality is missing you can't really see that tuple was the wrong choice for the data structure since it is immutable. If you tried to add another donor amount to existing donor you would have gotten this error: `AttributeError: 'tuple' object has no attribute 'append'`